lib/auth/webauthncli/api.go

```diff
@@ -138,14 +151,32 @@ func crossPlatformLogin(
 		return FIDO2Login(ctx, origin, assertion, prompt, opts)
 	}
 
+	if winwebauthn.IsAvailable() {
+		log.Debug("WIN_WEBAUTHN: Using windows webauthn for credential creation")
+		inOpts := &winwebauthn.LoginOpts{}
```

Comment:
Attachment here would always be cross-platform. Similarly in the method below.

Reply:
I will move that `webauthnwin.Login` above platform/crossplatform code so this will simplify